.github/workflows/e2e-test.yml

```diff
@@ -40,8 +40,24 @@ jobs:
         with:
           image: "kserve/storage-initializer:${{ github.sha }}"
 
+  poetry-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: Install Poetry and version plugin
+        run: ./test/scripts/gh-actions/setup-poetry.sh
+
+      - name: Check poetry lock file consistency
+        run: ./test/scripts/gh-actions/check-poetry-lockfile.sh
```

Comment:
tried out the script seems like kserve package is not consistent with the lock file
```
moving into folder .
skipping folder .
moving into folder ./xgbserver
~/github/kserve/python/xgbserver ~/github/kserve/python
poetry.lock is consistent with pyproject.toml.
~/github/kserve/python
moving into folder ./aiffairness
~/github/kserve/python/aiffairness ~/github/kserve/python
poetry.lock is consistent with pyproject.toml.
~/github/kserve/python
moving into folder ./.pytest_cache
skipping folder ./.pytest_cache
moving into folder ./lgbserver
~/github/kserve/python/lgbserver ~/github/kserve/python
poetry.lock is consistent with pyproject.toml.
~/github/kserve/python
moving into folder ./paddleserver
~/github/kserve/python/paddleserver ~/github/kserve/python
poetry.lock is consistent with pyproject.toml.
~/github/kserve/python
moving into folder ./plugin
skipping folder ./plugin
moving into folder ./kserve
~/github/kserve/python/kserve ~/github/kserve/python
Error: poetry.lock is not consistent with pyproject.toml. Run `poetry lock [--no-update]` to fix it.
```

Reply:
The github action is supposed to fail in this PR?

Reply:
Hmm... That's weird. For me the script runs successfully.